modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ViewConfigurationScreenMVCRenderCommand.java

```diff
@@ -57,6 +58,14 @@ public String render(
 			_configurationEntryRetriever.getConfigurationScreen(
 				configurationScreenKey);
 
+		if (!configurationScreen.isVisible()) {
+			throw new PortletException(
+				StringBundler.concat(
+					"The ", configurationScreen.getScope(), " configuration \"",
+					configurationScreen.getName(themeDisplay.getLocale()),
+					"\" is not accessible through the UI"));
```

Comment:
Hey @thiago-buarqque , this is looking good, but I'm unsure about the phrase "through the UI"? Why not just have "is not accessible" or "is not visible"? What do you think?

Reply:
@ethib137 that makes sense. I'll go with "is not accessible"